pldoc/static/js/pattern-library-doc.js

```diff
@@ -6,5 +6,7 @@ require(
         '../../_includes/examples/breadcrumbs-js.html',
         './edx-icons.js'
     ],
-    function($, Ui) {}
+    function() {
+        'use strict';
+    }
 );
```

Comment:
Nitpick: there should be a new line at the bottom of every file (edX code convention).